app/models/patient_import_row.rb

```diff
@@ -131,6 +137,10 @@ def date_of_birth
     nil
   end
 
+  def gender_code
+    @data["CHILD_GENDER"]&.strip&.downcase&.gsub(" ", "_") || "not_specified"
```

Comment:
```suggestion
    @data["CHILD_GENDER"]&.strip&.downcase&.gsub(" ", "_") || "not_known"
```

Should this default to `not_known` rather than not specified?

Reply:
Yes, looking at the definitions more closely, you're right. I've changed this.